dear_petition/petition/utils.py

```diff
@@ -48,12 +48,14 @@ def make_datetime_aware(dt_str):
     return aware_datetime_obj
 
 
-def format_petition_date(date):
+def format_petition_date(d):
     """ Format Date Objects for PDF Writer
 
     If date is true then the date object will be formatted
     according to DATE_FORMAT, else this defintion will return
     empty strings. Both of which can be written to the PDF
     correctly.
     """
-    return date.strftime(DATE_FORMAT) if date else ""
+    if isinstance(d, datetime):
+        d = dt_obj_to_date(d)
```

Comment:
PR #155 has a different solution for fixing the datetime. Is there one that we should choose over the other?

Reply:
Both will work! I'm okay deferring to #155, since I didn't have cycles to write the tests.